.orchestra/config/components/revng_js_prerequisites.yml

```diff
@@ -17,28 +17,16 @@ configure: |
   echo '(@= package_json_sha @) package.json' | sha256sum -c -
 
   #! Generate package lock
-  NPM=$(which -a npm | grep -v "$ORCHESTRA_ROOT" | head -n1)
-  "$NPM" install --package-lock-only --silent
-
-  #! from the package-lock.json read the urls of the tars we have to download
-  jq -r '.dependencies[].resolved' package-lock.json | \
-  while IFS= read -r URL; do
-    fetch.sh "$URL"
-  done
-
-  #! parse the integrity field in all depenendencies to check that the downloaded files match
-  #! the hash stored in package-lock.json
-  PACKAGE_NAMES=$(jq -r '.dependencies[].resolved' package-lock.json | sed 's;.*/\([^/]*\)$;\1;g')
-  PACKAGE_HASH=$(jq -r '.dependencies[].integrity' package-lock.json | sed 's;^sha512-;;' | while read -r HASH; do echo "$HASH" | base64 -d | xxd -p -c128; done)
-  paste -d' ' <(echo "$PACKAGE_HASH") <(echo "$PACKAGE_NAMES") | sha512sum -c -
+  "$ORCHESTRA_DOTDIR"/support/npm-wrapper/system-npm install --package-lock-only --silent
+  "$ORCHESTRA_DOTDIR"/support/js-populate-cache npm -o ./.cache
```

Comment:
Comment chicken and egg on `js-populate-cache` requiring `@yarnpkg/lockfile` (in theory).

Reply:
Done